types/logger.d.ts

```diff
@@ -0,0 +1,10 @@
+declare namespace Logger {
+  export function silly(): void;
+  export function debug(): void;
+  export function notice(): void;
+  export function info(): void;
+  export function warning(): void;
+  export function error(): void;
+}
+
+export = Logger;
```

Comment:
Should this be an interface?
```javascript
export interface Logger {
  silly(): void;
  debug(): void;
  notice(): void;
  info(): void;
  warning(): void;
  error(): void;
}
```

Reply:
It actually doesn't work because the export of the file is an object, not a class. So when one imports code from the file, they aren't importing an instance of the object, but rather the object itself. 

Interfaces work with instances. I _think_ namespaces work with these static objects. At any rate, I had errors when using an interface, and no errors when using the interface. I will do a bit more digging first though to ensure the type is most accurate before we resolve this.